apps/web/src/components/tx-flow/flows/SafenetTokenTransfers/CreateTokenTransfers.tsx

```diff
@@ -0,0 +1,169 @@
+import TokenIcon from '@/components/common/TokenIcon'
+import { SafeTxContext } from '@/components/tx-flow/SafeTxProvider'
+import TxCard from '@/components/tx-flow/common/TxCard'
+import commonCss from '@/components/tx-flow/common/styles.module.css'
+import { SafeAppsName } from '@/config/constants'
+import { useRemoteSafeApps } from '@/hooks/safe-apps/useRemoteSafeApps'
+import AddIcon from '@/public/images/common/add.svg'
+import { formatVisualAmount } from '@/utils/formatters'
+import { Alert, Button, CardActions, Divider, Grid, Link, SvgIcon, Typography } from '@mui/material'
+import { ZERO_ADDRESS } from '@safe-global/protocol-kit/dist/src/utils/constants'
+import { type TokenInfo } from '@safe-global/safe-gateway-typescript-sdk'
+import { useContext, useEffect, useState, type ReactElement } from 'react'
+import { FormProvider, useFieldArray, useForm } from 'react-hook-form'
+import { TokenTransfersFields, type TokenTransfersParams } from '.'
+import CSVAirdropAppModal from './CSVAirdropAppModal'
+import RecipientRow from './RecipientRow'
+
+export const AutocompleteItem = (item: { tokenInfo: TokenInfo; balance: string }): ReactElement => (
+  <Grid
+    container
+    sx={{
+      alignItems: 'center',
+      gap: 1,
+    }}
+  >
+    <TokenIcon logoUri={item.tokenInfo.logoUri} key={item.tokenInfo.address} tokenSymbol={item.tokenInfo.symbol} />
+
+    <Grid item xs>
+      <Typography
+        variant="body2"
+        sx={{
+          whiteSpace: 'normal',
+        }}
+      >
+        {item.tokenInfo.name}
+      </Typography>
+
+      <Typography variant="caption" component="p">
+        {formatVisualAmount(item.balance, item.tokenInfo.decimals)} {item.tokenInfo.symbol}
+      </Typography>
+    </Grid>
+  </Grid>
+)
+
+export const CreateTokenTransfers = ({
```

Comment:
We need a component to check that the total transfer amount is not higher than the users balance

Reply:
As we decided in our call, let's move this topic out of the happy path: https://github.com/safe-global/safenet/issues/290.
For now I implemented the UI alert, without any functionality.